lucene/benchmark/build.gradle

```diff
@@ -15,13 +15,13 @@
  * limitations under the License.
  */
 
-
-apply plugin: 'java-library'
+apply plugin: 'java'
+// NOT a 'java-library'.  Maybe 'application' but seems too limiting.
```

Comment:
I think java plugin is more than fine here so remove the comment for the final version, maybe?

Reply:
I like that this comment spells out a difference from how all the other modules are.

Reply:
From my (seasoned) gradle viewpoint this comment really doesn't make much sense: it's not an "application" in gradle sense - we launch multiple classes, have infrastructure in the build file, not a main class etc. But fine with me.